src/App.tsx

```diff
@@ -1,10 +1,51 @@
-import "./App.css";
+import { useState } from 'react';
+import { ThemeProvider } from 'styled-components';
+
+import CardNumberInput from './components/CardForm/CardNumberInput/CardNumberInput';
+import CardExpirationInput from './components/CardForm/CardExpirationInput/CardExpirationInput';
+import CardOwnerInput from './components/CardForm/CardOwnerInput/CardOwnerInput';
+import CardPreview from './components/CardForm/CardPreview/CardPreview';
+
+import theme from './styles/theme';
+import GlobalStyles from './styles/Global.style';
+import * as S from './styles/App.style';
 
 function App() {
+  const [cardNumbers, setCardNumbers] = useState(['', '', '', '']);
+  const [month, setMonth] = useState('');
+  const [year, setYear] = useState('');
+  const [owner, setOwner] = useState('');
+
+  const handleCardNumbers = (cardNumbers: string[]) => {
+    setCardNumbers(cardNumbers);
+  };
+
+  const handleMonth = (month: string) => {
+    setMonth(month);
+  };
+
+  const handleYear = (year: string) => {
+    setYear(year);
+  };
+
+  const handleOwner = (owner: string) => {
+    setOwner(owner);
+  };
+
   return (
-    <>
-      <h1>React Payments</h1>
-    </>
+    <ThemeProvider theme={theme}>
+      <GlobalStyles></GlobalStyles>
```

Comment:
self closing tag 사용 가능해보입니다ㅎㅎ
```suggestion
      <GlobalStyles/>
```

관련 eslint 룰 추가해도 괜찮을 것 같아요 :)